### PR TITLE
- Adds custom ImageAdmin with hack to show thumb on yourself.

### PR DESCRIPTION
**Pull request by seijihg**
Add custom menu to the admin panel so Images can be shorted via attributes. Will help content guys remove the big size images.

Default Image browser cant be used due to no option to extend its template to edit to change unless forking out wagtail repo itself and change the source code.

Screenshot 2021-11-12 at 21 00 12

No need to test since the image_edit_view and image_add_view are wagtails native apps.

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/GP2-3803
- [x] Jira ticket has the correct status.
- [x] [Changelog](CHANGELOG.md) entry added.

### Reviewing help

- [ ] Explains how to test locally, including how to set up appropriate data
- [x] Includes screenshot(s) - ideally before and after, but at least after

### Housekeeping

- [ ] Added all new environment variables to Vault.
- [ ] Cleaned up old feature flags
- [ ] Upgraded any vulnerable dependencies.
- [ ] Ran the `make manage download_geolocation_data` command
- [ ] I have updated security dependencies
- [ ] Python requirements have been re-compiled.
- [ ] Frontend assets have been re-compiled.

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

**Review by bobmeredith — COMMENTED** [review]
LGTM, but I think @webbyfox should review really as he's more wagtail savvy.

File: core/admin.py [hunk 1]
```diff
@@ -7,3 +14,53 @@
...
+        spec = self.thumb_image_filter_spec  # pragma: no cover
+        rendition = get_rendition_or_not_found(obj, spec)  # pragma: no cover
+        img_attrs.update({'src': rendition.url})  # pragma: no cover
+        return mark_safe(f'<img{flatatt(img_attrs)}>')  # pragma: no cover
+
+    edit_view_class = StandardImageEditViewWrapper
+    create_view_class = StandardImageCreateViewWrapper
+
+    def size(self, obj):
+        return f'{round(obj.file_size / 1024)}KB' if obj.file_size < 1048000 else f'{round(obj.file_size / 1048000)}MB'
```
Comment (webbyfox) [inline core/admin.py:63]: can you not get filesize from an in-built function like os.path.getsize?

File: core/admin.py [hunk 2]
```diff
@@ -7,3 +14,53 @@
...
+    add_to_settings_menu = False
+    exclude_from_explorer = False
+    list_display = (
+        'admin_thumb',
+        'title',
+        'alt_text',
+        'size',
+    )
+
+    def admin_thumb(self, obj):
```
Comment (webbyfox) [inline core/admin.py:46]: Do we really need a thumb?

File: core/admin.py (hunk 1 above)
Comment (webbyfox) [reply core/admin.py:63]: Also, some tests to verify this.

**Review by webbyfox — COMMENTED** [review]
Well done!!

File: core/admin.py (hunk 1 above)
Comment (seijihg) [reply core/admin.py]: I see no point if its exist in the database easier to pull it out from there :D

File: core/admin.py (hunk 2 above)
Comment (seijihg) [reply core/admin.py]: end of the day probably not :D removed!

**Review by webbyfox — APPROVED** [review]
LGTM

**Pull request merged**